saleor/plugins/webhook/tasks.py

```diff
@@ -481,25 +490,53 @@ def send_webhook_using_scheme_method(
     raise ValueError("Unknown webhook scheme: %r" % (parts.scheme,))
 
 
+def handle_webhook_retry(
```

Comment:
This could go to webhook/utils as well, no?

Reply:
@maarcingebala I didn't want to place this inside the utils, as inside `tasks.py` file we store everything related to sending the webhook and everything related to celery tasks. This function as an input accepts the celery task that should be re-tried. So personally I would prefer to keep it in `tasks.py` file. 
Let me know if you agree with this.

Reply:
Sure it's fine 👍